packages/esm-patient-chart-app/src/patient-chart/chart-review/dashboard-view.component.tsx

```diff
@@ -82,14 +67,19 @@ export function DashboardView({ dashboard, patientUuid, patient }: DashboardView
     <>
       <ExtensionSlot state={state} name="top-of-all-patient-dashboards-slot" />
       {!dashboard.hideDashboardTitle && resolvedTitle && <h1 className={styles.dashboardTitle}>{resolvedTitle}</h1>}
-      <ExtensionSlot
-        key={dashboard.slot}
-        name={dashboard.slot}
-        className={styles.dashboard}
-        style={{ gridTemplateColumns }}
-      >
-        <Extension state={state}>{wrapItem}</Extension>
-      </ExtensionSlot>
+      <div className={styles.dashboardContainer}>
+        <ExtensionSlot key={dashboard.slot} name={dashboard.slot} className={styles.dashboard}>
+          {(extension) => {
+            const { fullWidth = false } = widgetMetas[getExtensionNameFromId(extension.id)];
+            const style = fullWidth ? { gridColumn: '1 / -1' } : {};
```

Comment:
Rather than conditionally applying an element `style` attribute, it might be better to conditionally apply a classname, e.g.,

```ts
<div className={classNames(styles.extension, { [styles.gridLayout]: fullWidth })}
```

Reply:
my bad